Templates/CDS-CSharp/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Cds WebHook",
+    "description": "$cdsWebHookCSharp_description",
+    "defaultFunctionName": "CdsWebHookCSharp",
+    "language": "C#",
+    "category": [
```

Comment:
Change the value of category to `[ "$temp_category_experimental" ]`. This will get the template tagged as experimental.